src/PAModel/CanvasDocument.cs

```diff
@@ -81,6 +81,7 @@ public class CanvasDocument
 
         // Track all asset files, key is file name
         internal Dictionary<FilePath, FileEntry> _assetFiles = new Dictionary<FilePath, FileEntry>();
+        internal Dictionary<string, LocalAssetInfoJson> _localAssetInfoJson = new Dictionary<string, LocalAssetInfoJson>();
```

Comment:
Can you add a comment about what the key is, and also specify what comparison is being used (Case sensitive, insensitive, ...)